libtiledbsoma/test/unit_soma_dense_ndarray.cc

```diff
@@ -87,9 +87,6 @@ TEST_CASE("SOMADenseNDArray: basic") {
     REQUIRE(soma_dense->ctx() == ctx);
     REQUIRE(soma_dense->type() == "SOMADenseNDArray");
     REQUIRE(soma_dense->is_sparse() == false);
-    auto schema = soma_dense->schema();
```

Comment:
why the removes?

Reply:
The `schema` method now returns an `ArrowSchema` instead of a TileDB `ArraySchema`. I can change modify the test to reflect this instead of just removing it altogether.